features/all-positions/AllPositions.tsx

```diff
@@ -12,61 +12,114 @@ import {
 } from "@material-ui/core";
 import styled from "styled-components";
 import { utils } from "ethers";
+import { useState, useEffect } from "react";
 
 import EmpState from "../../containers/EmpState";
 import Collateral from "../../containers/Collateral";
 import Token from "../../containers/Token";
 import EmpSponsors from "../../containers/EmpSponsors";
-import EmpContract from "../../containers/EmpContract";
 import PriceFeed from "../../containers/PriceFeed";
 import Etherscan from "../../containers/Etherscan";
 
-import { getLiquidationPrice } from "../../utils/getLiquidationPrice";
-import { isPricefeedInvertedFromTokenSymbol } from "../../utils/getOffchainPrice";
-
-const fromWei = utils.formatUnits;
-
 const Link = styled.a`
   color: white;
   font-size: 18px;
 `;
 
+const ClickableText = styled.span`
+  &:hover {
+    cursor: pointer;
+    opacity: 0.6;
+  }
+  text-align: end;
+  user-select: none;
+`;
+
+const PageButtons = styled.div`
```

Comment:
I feel like there could be a bit more padding at the bottom here.
![image](https://user-images.githubusercontent.com/12886084/88589202-0dc67c80-d059-11ea-916d-78a1ead24d96.png)

Reply:
I set it from 0.5 to 2em